packages/3d-web-client-core/src/collisions/CollisionsManager.ts

```diff
@@ -45,10 +45,10 @@ export class CollisionsManager {
   public raycastFirstDistance(ray: Ray): number | null {
     let minimumDistance: number = Infinity;
```

Comment:
Just noticed that we don't actually return `null` if there isn't a distance. This could be type `number | null` and be set to `null`.

Reply:
Fixed in the [latest commit](https://github.com/mml-io/3d-web-experience/pull/24/commits/5666755670e32931fd5e041ffae592d384c22f40)